src/main/java/io/cdap/plugin/servicenow/source/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -89,14 +94,20 @@ public String getAccessToken() throws OAuthSystemException, OAuthProblemExceptio
     try {
       String accessToken = getAccessToken();
       requestBuilder.setAuthHeader(accessToken);
+      long start = System.currentTimeMillis();
       apiResponse = executeGet(requestBuilder.build());
+      long end = System.currentTimeMillis();
```

Comment:
start, end are now used. can be removedd

Reply:
`start`, `end` removed. 286b28256112b49eaab4a3a036a86973a3fa3bda